CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
 ## [Unreleased]
 
+## [2.0.1] - 2019-XX-YY
+
+### Changed
+
+- Migrated to Cloud Build. [#18]
```

Comment:
I don't think we need to have a new version, nothing is changed in the module itself, so no need for a release.

Reply:
Correct.

Reply:
@morgante, @averbuks! [I agree](https://github.com/terraform-google-modules/terraform-google-org-policy/pull/18#discussion_r331296479), you should probably double check migrated modules and revert unwanted release(if there is any).